RNSentryCocoaTester/RNSentryCocoaTesterTests/RNSentryOnDrawReporterTests.m

```diff
@@ -0,0 +1,16 @@
+#import <XCTest/XCTest.h>
+#import "RNSentryOnDrawReporter.h"
+
+@interface RNSentryOnDrawReporterTests : XCTestCase
+
+@end
+
+@implementation RNSentryOnDrawReporterTests
+
+- (void)testRNSentryOnDrawReporterViewIsAvailableWhenUIKitIs
+{
+  RNSentryOnDrawReporterView* view = [[RNSentryOnDrawReporterView alloc] init];
+  XCTAssertNotNil(view);
+}
```

Comment:
Strange test

Reply:
Yes, I just needed to use the class to make sure its available.